charts/keycloak/README.md

```diff
@@ -79,10 +79,8 @@ Parameter | Description | Default
 `keycloak.startupScripts` | Custom startup scripts to run before Keycloak starts up | `[]`
 `keycloak.lifecycleHooks` | Container lifecycle hooks. Passed through the `tpl` function and thus to be configured a string | ``
 `keycloak.extraArgs` | Additional arguments to the start command | ``
-`keycloak.livenessProbe.initialDelaySeconds` | Liveness Probe `initialDelaySeconds` | `120`
-`keycloak.livenessProbe.timeoutSeconds` | Liveness Probe `timeoutSeconds` | `5`
-`keycloak.readinessProbe.initialDelaySeconds` | Readiness Probe `initialDelaySeconds` | `30`
-`keycloak.readinessProbe.timeoutSeconds` | Readiness Probe `timeoutSeconds` | `1`
+`keycloak.livenessProbe` | Liveness probe configuration. Passed through the `tpl` function and thus to be configured a string | `httpGet:\n  path: {{ if ne .Values.keycloak.basepath "" }}/{{ .Values.keycloak.basepath }}{{ end }}/\n  port: http\ninitialDelaySeconds: 120\ntimeoutSeconds: 5`
```

Comment:
```suggestion
`keycloak.livenessProbe` | Liveness probe configuration. Passed through the `tpl` function and thus to be configured as string | `httpGet:\n  path: {{ if ne .Values.keycloak.basepath "" }}/{{ .Values.keycloak.basepath }}{{ end }}/\n  port: http\ninitialDelaySeconds: 120\ntimeoutSeconds: 5`
```

Not sure if we should specify the full defaults here because the `\n` doesn't render as line-break. I'd probably just refer to the `values.yaml`:  "See `values.yaml` for defaults."